addons/hr_skills/static/src/fields/resume_one2many/resume_one2many.js

```diff
@@ -30,7 +30,14 @@ ResumeListRenderer.rowsTemplate = "hr_skills.ResumeListRenderer.Rows";
 ResumeListRenderer.recordRowTemplate = "hr_skills.ResumeListRenderer.RecordRow";
 
 
-export class ResumeX2ManyField extends SkillsX2ManyField {}
+export class ResumeX2ManyField extends SkillsX2ManyField {
+    setup() {
+        this._openRecord = (params) => {
+            params.title = _t("Create a resume line");
+            openRecord({...params});
```

Comment:
It's the same on the original component but isn't that basically the same? :thinking: 
```suggestion
            openRecord(params);
```

Reply:
Yes indeed, the only difference I can think of is that {...params} copies the object, instead of just passing it on, so that changes downstream are not recorded in the params object upstream, but I don't really see how that would matter here.

Reply:
Changed it